### PR TITLE
Bump the version to 1.4.0

### PR DESCRIPTION
**Pull request by kienstra**
#### Changes
* Bumps the version to `1.4.0` to prepare for the release on Thursday, October 14th
* Adds a CHANGELOG entry.
* Bumps the minimum WP version to `5.7`

#### Testing instructions
1. `php bin/verify-versions.php`
2. Expected: 'Success! All of the version numbers are the same, and there's a CHANGELOG.md entry.'

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 ## Changelog ##
 
+### 1.4.0 - 2021-10-14 ###
+
+Add a modal option to make longer blocks easier, fix for widgets.php
```
Comment (johnstonphilip) [inline CHANGELOG.md]: Maybe missing a `*` at the start of this line?

Comment (kienstra) [reply CHANGELOG.md]: Good catch, I deleted that line in https://github.com/studiopress/genesis-custom-blocks/pull/95/commits/060cee38b26163893d4790da2901d1848fe461ad. 

That was intended to be a summary. But there are only 2 bullet points below it. No summary needed 😄

Comment (johnstonphilip) [reply CHANGELOG.md]: ah I see! Sorry I'm a bit new to proper changelog formatting :D

**Review by johnstonphilip — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kienstra) [reply CHANGELOG.md]: Oh all good, it looked awkward having a summary almost as long as the 2 bullet points combined.

**Review by dreamwhisper — APPROVED** [review]

**Comment by kienstra** [comment]
If it's alright, I removed the line about the `widgets.php` fix, as https://github.com/studiopress/genesis-custom-blocks/pull/94 isn't ready.

**Pull request merged**